app/helpers/products_helper.rb

```diff
@@ -46,20 +48,20 @@ def public_calendar_options(product)
     if current_facility&.show_instrument_availability? || !current_facility && product.facility.show_instrument_availability?
       public_calendar_availability_options(product)
     else
-      { class: ["fa fa-calendar fa-2x"],
+      { class: ["fa fa-calendar fa-lg fa-fw"],
         title: t("instruments.public_schedule.icon") }
     end
   end
 
   def public_calendar_availability_options(product)
     if product.offline?
-      { class: ["fa fa-calendar fa-2x", "in-use"],
+      { class: ["fa fa-calendar fa-lg fa-fw", "in-use"],
```

Comment:
I feel like there might be a way to dry up all these shared class lists.

Reply:
Probably, but literally everywhere we make calls to font-awesome it looks like this, so I think it might make more sense just to live with this. Especially if we're going to end up changing everything again when font-awesome-rails moves to font-awesome 5.

Reply:
That kind of why I was thinking about DRYing it up, so it would ideally be simpler to upgrade to fa5. I'm fine if we do the cleanup at that point.